### DISCUSSION TITLE
Experimental Coref

### DISCUSSION THREAD

**Discussion by radandreicristian** [explosion/spaCy#11545] [Help: Coding & Implementations] [answered]
Hello,

I want to use the `spacy-experimental` coreferencing module that was merged today by @polm .

Is it possible to load a ready-to-use model for coreferncing, or do I have to train it myself? If the first one is possible, it is still very unclear from the documentation how to do it.

If it's not possible, will it be possible to have a pre-trained module ready to use in the future?

Thanks:)

**Comment by polm** [comment, marked as answer]
Hey, we just made [a release with this feature](https://github.com/explosion/spacy-experimental/releases/tag/v0.6.0). It includes a pretrained model you can use right away, and the [example project](https://github.com/explosion/projects/tree/v3/experimental/coref) shows how to train it. 

Keep in mind this is still experimental, so if you run into any issues or unclear things, just let us know.

**Comment by svlandeg** [comment]
To add to Paul's answer, you can also see a very quick overview of how to use the pretrained coref model in this thread: https://github.com/explosion/spaCy/discussions/11585
